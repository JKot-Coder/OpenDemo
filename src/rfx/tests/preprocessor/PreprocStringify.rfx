// :PREPROCESSOR

#define A a
#define B b

// Correct output
// "A B"

#define STRINGIFY(x) #x
STRINGIFY(A B);