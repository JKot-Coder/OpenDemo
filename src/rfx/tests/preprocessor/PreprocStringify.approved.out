SourceOutput:[
    #line 10 "../src/rfx/tests/preprocessor/PreprocStringify.rfx"
    "A B";
]
Diagnostic:[
]
